vnext/proposals/active/keyboard-reconcile-desktop.md

```diff
@@ -68,6 +68,7 @@ Where `IKeyboardEvent` is a new event type added to `ReactNative.NativeSynthetic
 | shiftKey | boolean | The Shift key. | false |
 | metaKey | boolean | Maps to Windows Logo key and the Apple Command key. | False |
 | repeat | boolean | Flag to represent if a key is being held down/repeated. Tracked by [Windows Issue#5513](https://github.com/microsoft/react-native-windows/issues/5513), [macOS Issue#502](https://github.com/microsoft/react-native-macos/issues/502) | False |
+| timestamp | number | The time, relative to the system boot time, in milliseconds. | undefined |
```

Comment:
Do you know if this also exist on web key events?

Reply:
I'm assuming so: https://developer.mozilla.org/en-US/docs/Web/API/Event/timeStamp

There's a discrepency between `timestamp` and `timeStamp`, but I'd rather be consistent with the RN GestureResponderEvent than with Web for this.

Reply:
Seems like even RN is inconsistent on `timestamp` vs. `timeStamp`.

Reply:
Grr, that's annoying. Any sense of which is used more in stock RN?

Reply:
Looks like `timeStamp` is added to all JS events:
https://github.com/facebook/react-native/blob/dc80b2dcb52fadec6a573a9dd1824393f8c29fdc/Libraries/Types/CoreEventTypes.js#L31
https://github.com/facebook/react-native/blob/0afd71a18d19edef3d6603cabd2672ffde5ba30e/Libraries/Components/DatePicker/DatePickerIOS.ios.js#L136

It looks like if the event object includes a `timeStamp` property, it will be preferred over Date.now(). Probably only react-native-web uses this.

Change events for DatePicker use `timestamp`:
https://github.com/facebook/react-native/blob/0afd71a18d19edef3d6603cabd2672ffde5ba30e/Libraries/Components/DatePicker/DatePickerIOS.ios.js#L136

And iOS and Android use `timestamp` in pointer events.

It feels like the "convention" is to use `timestamp` on `nativeEvent`, so this approach is probably best for now, and we should try to keep things consistent between pointer and keyboard events.

I think there's a broader discussion about whether we should figure out a way to emit a timestamp more broadly so it can be lifted to the event in JS via the `timeStamp` property, but let's not solve that here.

Reply:
I think this prop should be added to the type definition of KeyEvent in CoreEventTypes.windows.js